src/components/find-on-shelf/FindOnShelfModal.tsx

```diff
@@ -78,15 +78,38 @@ const FindOnShelfModal: FC<FindOnShelfModalProps> = ({
   });
 
   // Sorting of the data below to show branches & manifestations in the correct order.
```

Comment:
Consider elaborating what the "correct" order is in a comment to help provide an overview of the following code:

1. Main library branch first
2. Library branches sorted alphabetically with available holdings second
3. Library branches sorted alphabetically without available holdings third

You could even consider moving sorting into a separate function to reduce the complexity of the component code itself.

Reply:
Very good call with more precise comment. 
I was toying with the idea of throwing it out of the component, but then I ended up with the thought that it's so specific to this component that I think it would make sense to keep it in.